jboss/container/wildfly/launch-config/os/added/launch/jboss_modules_system_pkgs.sh

```diff
@@ -1,19 +0,0 @@
-#!/bin/sh
```

Comment:
@jamezp , I think that we should keep this script. The JBOSS_MODULES_SYSTEM_PKGS_APPEND is documented.
In addition, for JDK8 image (EAP/XP), com.sun.crypto.provider is expected to be a system pkg, and that is the way it is conveyed to the server configuration (although we could set it only for JDK8 builder image).
Would it be possible to keep the script and remove org.jboss.logmanager from its default value (I guess that it is the reason of the removal)?

Reply:
@jfdenise Yes. The only reason I removed it was I didn't think it would be required since the log manager no longer needs to be on the boot class path.